test/test_isl.py

```diff
@@ -71,6 +71,22 @@ def test_subst_into_pwqpolynomial():
     assert (result - expected_pwqpoly).is_zero()
 
 
+def test_subst_into_pwaff():
+    from pymbolic.primitives import Variable
+    arg_dict = {
+            "m": 3*Variable("nx"),
+            "n": 2*Variable("ny")}
+    space = isl.Set("[nx, ny, nz] -> { []: }").params().space
+    poly = isl.PwAff("[m, n] -> { [3 * m + 2 * n] : "
+        "m > 0 and n > 0; [7* m + 4*n] : m > 0 and n <= 0 }")
+
+    from loopy.isl_helpers import subst_into_pwaff
+    result = subst_into_pwaff(space, poly, arg_dict)
+    expected = isl.PwAff("[nx, ny, nz] -> { [(9nx + 4ny)] : nx > 0 and ny > 0;"
+            " [(21nx + 8ny)] : nx > 0 and ny <= 0 }")
+    assert result == expected
```

Comment:
Another possible way to test this would be to convert to pymbolic expression, substitute there, evaluate both for some concrete integer inputs, and make sure they're the same.

Reply:
I think this way is more general. Didn't implement it.